tubemq-client-twins/tubemq-client-cpp/include/tubemq/tubemq_config.h

```diff
@@ -34,6 +34,42 @@ using std::set;
 using std::string;
 
 
+// configure for log, thread pool etc.
+class TubeMQServiceConfig {
+ public:
+  TubeMQServiceConfig();
+  ~TubeMQServiceConfig();
+  TubeMQServiceConfig& operator=(const TubeMQServiceConfig& target);
+  void setLogCofigInfo(int32_t log_max_num,
```

Comment:
The public function first letter needs to be capitalized.

Reply:
Ok